### DISCUSSION TITLE
Select all inside raw string literal

### DISCUSSION THREAD

**Discussion by AntonC9018** [JetBrains/ideavim#714] [Q&A]
I'm not aware of the proper terminology, but like you can do `vi"` to select everything within a string literal. There should be that for [raw string literals in C#](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-11.0/raw-string-literal). Can you share the plugin that adds that if it exists?

**Comment by citizenmatt** [comment]
There is nothing built in. The current implementation of the `i"` text object is based on Vim's implementation, not based on the IDE's knowledge of what a string literal is. That is, it looks for something surrounded by double quotes. Do you know if there are any Vim plugins that support C# raw string literals? I didn't see any from a quick look, but that could help - it might be possible to use that as part of your `.ideavimrc`. 

Alternatively, it would be possible to create a Kotlin extension that adds a new text object, and that could possibly use the IDE's knowledge to select the appropriate text (however, Rider's architecture might make this tricky, as raw string literals are rather complex, and the information might not be available to IdeaVim. Additionally, IdeaVim might not have access to the appropriate data when run in remote development scenarios)
